libs/dh/shared/domain/project.json

```diff
@@ -16,35 +16,48 @@
     "lint": {
       "executor": "@nx/eslint:lint"
     },
-    "generate-no-deps": {
+    "generate": {
       "executor": "nx:run-commands",
-      "cache": true,
+      "cache": false,
       "inputs": [
         "{workspaceRoot}/codegen.ts",
-        "{workspaceRoot}/libs/dh/shared/feature-graphql-codegen/**/*.js",
+        "{workspaceRoot}/libs/dh/shared/util-apollo/src/lib/codegen.js",
```

Comment:
question: Is this right? I don't see a `util-apollo/src/lib/codegen.js` file and I don't recall having created it myself. I also think that `{workspaceRoot}/libs/dh/shared/feature-graphql-codegen/**/*.js` should still be in the output (it is what generates the `ApolloDataSource` stuff.

Reply:
Mistake I would say. Have reverted